data/hai missions.txt

```diff
@@ -1053,3 +1053,251 @@ mission "Hai Honeymoon"
 	on complete
 		payment 100000
 		dialog `Instead of landing in the spaceport where they can be seen, you drop Anaya and Touhar off as close as you can to where they will be staying. The house is a few kilometers away, but they assure you that they can make it on their own. They pay you <payment> and thank you for bringing them this far.`
+
+
+
+mission "Nanachi 1"
+	name "Escort Nanachi to Cloudfire"
+	description "Teach Nanachi, a Hai who just bought her first ship, the basics of being a starship captain."
+	source "Giverstone"
+	destination "Cloudfire"
+	deadline
+	to offer
+		"net worth" > 20000000
+		"combat rating" > 1200
+		"random" > 85
+	npc accompany save
+		personality escort timid
+		government "Hai"
+		ship "Heavy Shuttle" "Acorn Chaser"
+	on visit
+		dialog "You have reached <planet>, but you left Nanachi's ship behind! Better depart and wait for them to arrive in this star system."
+	on offer
+		conversation
+			"As you are making your way through the spaceport you notice a worried-looking Hai staring at the local job board and back at some sort of document they are holding."
+			choice
+				"(Approach the Hai.)"
+				"(Leave them alone.)"
```

Comment:
Again: backticks, not quotes. Also, these should be tabbed in within the quotes.

```
choice
	`	(Approach the Hai.)`
	`	(Leave them alone.)`
```